internal/backends/winit/lib.rs

```diff
@@ -141,6 +139,12 @@ pub struct Backend {
         )
             -> Result<(Box<dyn WinitCompatibleRenderer>, Rc<winit::window::Window>), PlatformError>,
     event_loop_state: std::cell::RefCell<Option<crate::event_loop::EventLoopState>>,
+
+    /// This hook is called before a Window is created.
+    ///
+    /// It can be used to adjust settings of window that will be created
```

Comment:
Perhaps the docs should give an example _how_ to use this?

I'm not entirely sure myself. Is it like this? (untested!)

```
slint::platform::set_platform(Box::new({
    let mut backend = Backend::new().unwrap();
    backend.window_builder_hook = Some(Box::new(|builder| builder.with_foo()));
    backend
}));
```

If the docs have it, then that would also serve as compile test :)

Reply:
Updated.  Also fixed wasm